src/net/sourceforge/kolmafia/persistence/ItemDatabase.java

```diff
@@ -2205,6 +2205,17 @@ public static void parseCleaver(final String desc) {
     }
   }
 
+  public static void parseDesignerSweatpants(final String desc) {
+    Pattern sweatPattern = Pattern.compile("Your sweatpants are currently (\\d+)% sweat-logged.");
```

Comment:
Ah no this is not exactly what I meant - keep the consequences.txt bit and just make this bit say -> if we can't find "Your sweatpants are currently" set it to zero.

Reply:
My suggestion is slightly better in terms of pattern compilation I think.